x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -177,6 +178,7 @@ export class IngestManagerPlugin
         id: PLUGIN_ID,
         name: 'Ingest Manager',
         icon: 'savedObjectsApp',
+        category: DEFAULT_APP_CATEGORIES.security,
```

Comment:
Looks like this was a typo?
```suggestion
        category: DEFAULT_APP_CATEGORIES.management,
```